src/k8s-frameworkcontroller/deploy/delete.sh.template

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+pushd $(dirname "$0") > /dev/null
+
+ENDPOINT={{ cluster_cfg['layout']['kubernetes']['api-servers-url'] }}/apis/frameworkcontroller.microsoft.com/v1/namespaces/default/frameworks/
```

Comment:
will give you the approach to delete crd instead of frameworks

Reply:
anyway it will delete all jobs, is it too dangerous?
Seems delete controller should just delete controller instead delete its managed objects?

Reply:
current yarn launcher will also delete all jobs in [service delete](https://github.com/microsoft/pai/blob/master/src/yarn-frameworklauncher/deploy/delete.sh#L27), just keep the consistent

Reply:
If so, delete this:
`https://10.169.8.232:8443/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions/frameworks.frameworkcontroller.microsoft.com`

Reply:
Or  kubectl delete customresourcedefinitions frameworks.frameworkcontroller.microsoft.com

Reply:
updated